### ISSUE TITLE
Adding custom message on log-in

### ISSUE THREAD

**Issue by aliceingame** [Deadrik/TFCraft#542]
I updated the translation file to the latest version of google spreadsheet and I can easily translate everything in any language automatically using google translate. I would use it if it's possible to put a chat message on log-in that would tell people that the translation is made by google translate and that they can help improve the translation by visiting the translation tread on the forum or a wiki page.
That line could be used for announcements in English or stay empty.

**Comment by Kittychanley** [comment]
For TFC2, it's very likely that we're going to stop official localization support entirely, and just have it handled through resource packs.

**Comment by aliceingame** [comment]
I do not see why.
For me it's a bit rude to foreign language speakers and to the people who spend time translating the game.
Why would non English users have to do extra steps in order to play in their language ?
If you do so, will the TFC staff handle an official ressource pack for the game translation ?
I understand that localization and improvement of display of the text in foreign languages is not a priority, but I think that this is going a bit too far... especially when then only thing you have to do to maintain the localization in the release is to accept my pull request.

**Comment by Kittychanley** [comment]
Because we currently have no way of checking that the current translations are correct. I'm more comfortable removing the "officially supported" tag from things that we have no way of monitoring for actually being correct and appropriate.

**Comment by Kittychanley** [comment]
You are more than welcome to create a composite language resource pack with all of the languages on it and post it on the forums, it just won't be in a post officially endorsed by the devs, or actually included in the base mod.

**Comment by aliceingame** [comment]
Translations must be easy to get for players, if it's in a resource pack, it must be clearly visible in the download section or in a pinned post on top of the forum.

I don't know if you ever had an experience of having to deal with a foreign language environment, but it can already be challenging for non English speakers to find things on an English based website. To officially offering translation is only showing respect toward all of you non-English speaking players.

If the people have no idea the game is translated in their languages and that they have no easy way to find the right resource pack, they might just give up and stop playing the game after a while.

Officially supporting localization is to show respect toward non-English speakers.

I do not understand what is the problem if it's official. What are you so afraid of ? What if a translation is not accurate ? People will tell you and ask for it to be changed.... that's it.

If you do not officially support translation of the game, then you should at least give easy direction to non-English speakers on how to find translation in their language. For example a link in the official download page to the "unofficial" resource pack, with a message in every language available to say "to play TFC in an other language, please download the non-official resource pack. It is not official and not endorsed by the developers"... of course I would let you personally translate it or pay for professional translation service.

**Comment by Deadrik** [comment]
Kitty is correct that going the resource pack route is probably the best for us as a small team. It will alleviate some of our workload and make it easier for you to update new translations as they occur. So here is a compromise. You create a resource pack and release it on the forums and manage contributors there. On our download page, we will create a link that says "For translation to <insert language here> please check the unofficial language pack." Then we will have that link in as many languages as needed to make it easier for players to find. The only thing that I need from you is to have that phrase translated into a bunch of languages.

This way, you can update whenever it is appropriate without waiting on us to check the PRs, and we don't have to deal with the translations which we cant verify ourselves. It's a win/win.
